dace/frontend/fortran/ast_transforms.py

```diff
@@ -263,55 +1065,83 @@ def visit_Call_Expr_Node(self, node: ast_internal_classes.Call_Expr_Node):
             self.count = self.count + 1
         tmp = self.count
 
-        for i, arg in enumerate(node.args):
-            # Ensure we allow to extract function calls from arguments
-            node.args[i] = self.visit(arg)
+        # for i, arg in enumerate(node.args):
```

Comment:
please clean up commented code. it's unclear if it's an old remnant or something that is yet to be uncommented.